doc/nrf/app_dev/device_guides/nrf54l/index.rst

```diff
@@ -69,4 +69,3 @@ Ensure to check the revision of your nRF54L15 device to see if it is supported:
    testing_dfu
    vpr_flpr
    building_nrf54l
-   snippets
```

Comment:
Additionally, on this page the references to snippets have to be removed (lines 29 and 34)

Reply:
We could keep this as text for now, and once doc pages are available in the down stream - https://github.com/zephyrproject-rtos/zephyr/pull/82298, we could add this link.

Reply:
Thanks @annwoj and @divipillai, please take another look now.